frontend/src/common/HelpDialog.tsx

```diff
@@ -0,0 +1,37 @@
+import HelpOutlineIcon from '@mui/icons-material/HelpOutline'
+import { Button, Dialog, DialogContent, DialogTitle, Slide, Typography } from '@mui/material'
+import { TransitionProps } from '@mui/material/transitions'
+import React, { ReactNode } from 'react'
+
+const Transition = React.forwardRef(function Transition(
+  props: TransitionProps & {
+    children: React.ReactElement<any, any>
+  },
+  ref: React.Ref<unknown>,
+) {
+  return <Slide direction='up' ref={ref} {...props} />
+})
+
+type Props = {
+  infoLabel: string
```

Comment:
Consider renaming to `label`

Reply:
Removed as we no longer need it due to using the IconButton component instead of a regular Button